server_environment/__manifest__.py

```diff
@@ -31,6 +31,4 @@
     "data": [
         'serv_config.xml',
     ],
-    'installable': False,
```

Comment:
I think we should not remove let the `installable` key, set it to `True` instead.